examples/kubernetes/README.md

```diff
@@ -0,0 +1,6 @@
+# Distributed Optimization on Kubernetes
+
+This folder contains two kinds of examples with Kubernetes: one is based on `sklearn_simple.py` and the other is based on `pytorch_lightning_simple.py` with MLflow.
+
+Currently, both `./simple/sklearn_distributed.py` and `./mlflow/pytorch_lightning_distributed.py` use POSTGRESQL for their backend of `optuna.Study.optimize` to be parallelized.  
```

Comment:
Some of these file references could be actual links. Ditto for similar occurrences in other READMEs.